documentation/docs/extending/datasource.mdx

```diff
@@ -61,12 +55,9 @@ class CdfModelIdentifier implements ModelIdentifier {
 | `getModelOutputs()`     | Yes                                        | Returns an object describing the set of valid formats and version for the given model |
 
 `getModelUri()` returns a "base URL" that will be passed to `ModelDataProvider.getBinaryFile()` and `getJsonFile()`. Note that
-its up to `ModelDataProvider` how this base URL is handled for CAD models and it is possible to use this base URL as 
-some other identifier, e.g. a folder on disk or ID of a blob storage. For point clouds, however, this
-needs to be an actual URL as we base point cloud streaming on a third party library which manages it's own
-network layer. 
-It will require an instance of `BlobOutputMetadata` as input which defines the format, version, and if 
-applicable, the remote blob id of the output.
+its up to `ModelDataProvider` how this base URL is handled and it is possible to use this base URL as 
```

Comment:
```suggestion
it's up to `ModelDataProvider` how this base URL is handled and it is possible to use this base URL as 
```

Reply:
Nope, "it's" -> "it is", "its" is possessive, see https://www.dictionary.com/e/its-vs-its/